### DISCUSSION TITLE
Flag "docker" not defined while installing the project

### DISCUSSION THREAD

**Discussion by amelielecoz** [symfony/symfony#45823] [Symfony Book: The Fast Track] [answered]
Hi there, I ran the installation command 
`symfony new guestbook --version=6.0 --php=8.1 --webapp --docker --cloud`
as mentionned in this page :
https://symfony.com/doc/6.0/the-fast-track/fr/3-zero.html#initialiser-le-projet

My Symfony CLI version is 4.28.1 

But I get this error :

`Incorrect usage: flag provided but not defined: -docker  `

Indeed "--docker" is not listed in the possible options. Is there a way to go around this error and still install the project properly ?
I removed the --docker option and the project is installed, but will this cause any further errors ?

**Comment by MrYamous** [comment, marked as answer]
Hi, 

The `docker` flag has been added in the release 5.2, [according to changelog](https://github.com/symfony-cli/symfony-cli/releases/tag/v5.2.0). 
If you have opportunity to upgrade your Symfony CLI it should work fine

**Reply by amelielecoz** [reply]
Thank you MrYamous, indeed, with Symfony CLI 5.2, it works fine !

**Comment by Nicolasjmcrt** [comment]
Hello Amelie. J'ai le même problème que tu as rencontré. Peux-tu stp me dire comment tu as procédé ? Merci d'avance et bonne journée à toi.

**Reply by amelielecoz** [reply]
Bonjour Nicolas, mon Symfony CLI n'était pas à jour (version 4.2 de mémoire), je l'ai simplement mis à jour vers la version 5.2.
